tests/unit/test_subscription_request.py

```diff
@@ -346,3 +346,23 @@ def test_toar_tool_success():
 
     expected = {"type": "toar", "parameters": {"scale_factor": 12345}}
     assert res == expected
+
+
+def test_pv_source_success(geom_geojson):
+    """Configure a planetary variable subscription source."""
+    # NOTE: this function does not yet validate type and id.
+    # The nonsense values are intended to fail when the function does
+    # add validation.
+    source = subscription_request.planetary_variable_source(
+        "var1",
+        "VAR1-abcd",
```

Comment:
Would you be opposed to having real types and IDs here? That way:
1. when someone reads the test they can have an example of working code
2. the test will already pass when we do implement validation of types and ids.

I do appreciate keeping things simple, but I wonder if this small change is worth it.
```suggestion
        "soil_water_content",
        "SWC-AMSR2-C_V1.0_100",
```

Reply:
@kevinlacaille I get what you mean, but I'm intending that this test should fail if/when we do have some validation. Maybe the thing to do is both.

Reply:
Makes sense to me!